package.json

```
@@ -42,20 +40,21 @@
   },
   "homepage": "https://github.com/mojaloop/ml-testing-toolkit-shared-lib#readme",
   "dependencies": {
-    "ajv": "8.5.0",
+    "ajv": "8.11.0",
     "faker": "5.5.3",
-    "json-schema-faker": "0.5.0-rcv.34",
-    "json-schema-ref-parser": "9.0.7",
+    "json-schema-faker": "0.5.0-rcv.44",
+    "json-schema-ref-parser": "9.0.9",
     "lodash": "4.17.21",
     "node-dir": "0.1.17"
   },
   "devDependencies": {
-    "@types/jest": "26.0.23",
-    "jest": "27.0.4",
-    "jest-junit": "12.1.0",
-    "npm-audit-resolver": "2.3.1",
-    "npm-check-updates": "11.6.0",
-    "standard": "^16.0.3"
+    "@types/jest": "28.1.6",
+    "audit-ci": "^6.3.0",
+    "jest": "28.1.3",
+    "jest-junit": "14.0.0",
+    "npm-check-updates": "15.3.4",
+    "pre-commit": "^1.2.2",
+    "standard": "^17.0.0"
```

Please add `standard-version` `postchangelog` config as follows:

```suggestion
    "standard": "^17.0.0"
  },
  "standard-version": {
      "scripts": {
          "postchangelog": "replace '\\[mojaloop/#(\\d+)\\]\\(https://github.com/mojaloop/(.*)/issues/(\\d+)\\)' '[mojaloop/#$1](https://github.com/mojaloop/project/issues/$1)' CHANGELOG.md"
      }
```

NOTE:
- you will need to run `npm I -D replace` to ensure the dependency is met.
- please double check the indentation here